### PR TITLE
Blog post for 20.0.0.9 beta

### PR DESCRIPTION
**Pull request by lauracowen**
How do you think this reads?

Preview at https://draft-openlibertyio.mybluemix.net/blog/2020/08/05/jakarta-grpc-beta-20009.html

Please leave any comments in this PR.

### CODE REVIEW

**Comment by lauracowen** [comment]
I will also remove the duplicate gRPC material from last month's GA blog post and point at this one instead, I think.

File: posts/2020-08-05-jakarta-grpc-beta-20009.adoc [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+= More Jakarta EE 9 features and gRPC support in Open Liberty 20.0.0.9 beta
+Laura Cowen <https://github.com/lauracowen>
+:imagesdir: /
+:url-prefix:
+:url-about: /about/
+
+We now have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+* <<allbeta, All Beta Features>>: a new, larger package that contains all our non-Jakarta EE 9 beta features, any other Open Liberty features that are necessary for them to run, _and_ the Jakarta EE 9 features.
```
Comment (mbroz2) [inline posts/2020-08-05-jakarta-grpc-beta-20009.adoc]: @cbridgha can provide a definitive answer since he's the one doing the packaging, but I believe this is incorrect.  I believe "..that contains all Open Liberty beta features and functions plus our GA [or "non-beta" or "regular"] features and [non-conflicting] functions" would be more accurate

File: posts/2020-08-05-jakarta-grpc-beta-20009.adoc [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+* JTA 2.0 (`transaction-2.0`)
+* JSON-B 2.0 (`jsonb-2.0`)
+* JSON-P 2.0 (`jsonp-2.0`)
+* Servlet 5.0 (`servlet-5.0`)
+* JSP 3.0 (`jsp-3.0`)
+* Expression Language 4.0 (`el-4.0`)
+
+=== Try it now
+
+To try out these Jakarta EE 9 features on Open Liberty in a lightweight package, just update your build tools to pull the Open Liberty Jakarta EE 9 Beta Features package instead of the main release. The beta works with Java SE 11 or Java SE 8.
```
Comment (mbroz2) [inline posts/2020-08-05-jakarta-grpc-beta-20009.adoc]: I take it there's something that currently prevents Jakarta EE 9 from being compatible with Java SE 14?

Comment (kwsutter) [reply posts/2020-08-05-jakarta-grpc-beta-20009.adoc]: Not aware of anything... I'd reword this to say something like "The beta works with Java SE 8 or later."

File: posts/2020-08-05-jakarta-grpc-beta-20009.adoc [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+* <<allbeta, All Beta Features>>: a new, larger package that contains all our non-Jakarta EE 9 beta features, any other Open Liberty features that are necessary for them to run, _and_ the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools (either the Jakarta EE 9 Beta Features package or the All Beta Features package, depending on what you want to try).
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+The main change visible to developers in the Jakarta EE 9 planned release is the names of packages changing to accomodate the new `jakarta.*` namespace. In this Open Liberty beta, we have more Jakarta EE 9 features with their namechange completed. We've also made good TCK progress with these features.
```
Comment (kwsutter) [inline posts/2020-08-05-jakarta-grpc-beta-20009.adoc]: Is "namechange" one word or two?  Maybe it depends on which side of the pond you are on... ;-)

Comment (kwsutter) [reply posts/2020-08-05-jakarta-grpc-beta-20009.adoc]: I would also say "excellent TCK progress" instead of "good TCK progress".  Or, if that's too optimistic for you, how about "solid TCK progress"?  Good just sounds too average and I think we're doing a fine job with TCK testing.

File: posts/2020-08-05-jakarta-grpc-beta-20009.adoc [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-jakartaee9', version: '[20.0.0.9-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+Enable the Jakarta EE 9 beta features in your app's `server.xml`. You can enable the individual features you want (but remember if you enable the `jsp-3.0` feature, the `servlet-5.0` and `el-4.0` features are automatically enabled for you) or you can just add the Jakarta EE 9 convenience feature to enable them all at once:
```
Comment (kwsutter) [inline posts/2020-08-05-jakarta-grpc-beta-20009.adoc]: Use the convenience features to "enable all beta features at once".  (See my next comment as well.)

File: posts/2020-08-05-jakarta-grpc-beta-20009.adoc [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+
+[source, xml]
+----
+<server>
+  <featureManager>
+    <feature>jakartaee-9.0</feature>
+  </featureManager>
+</server>
+----
+
```
Comment (kwsutter) [inline posts/2020-08-05-jakarta-grpc-beta-20009.adoc:88]: Can we also add the web profile feature to show that it's available?

```
<server>
  <featureManager>
    <feature>webProfile-9.0</feature>
  </featureManager>
</server>
```

**Review by kwsutter — APPROVED** [review]
Just a few comments that should be addressed before publishing, but overall it looks solid enough for Approval.  Thanks, @lauracowen!

File: posts/2020-08-05-jakarta-grpc-beta-20009.adoc (hunk 2 above)
Comment (lauracowen) [reply posts/2020-08-05-jakarta-grpc-beta-20009.adoc]: Changed to specify the three versions.

File: posts/2020-08-05-jakarta-grpc-beta-20009.adoc (hunk 3 above)
Comment (lauracowen) [reply posts/2020-08-05-jakarta-grpc-beta-20009.adoc]: - "namechange" - just a typo, now fixed. :) 
- TCK progress is now "excellent" instead of mediocre. Even as a Brit I can gather a bit of optimism... ;)

File: posts/2020-08-05-jakarta-grpc-beta-20009.adoc (hunk 4 above)
Comment (lauracowen) [reply posts/2020-08-05-jakarta-grpc-beta-20009.adoc]: Changed.

File: posts/2020-08-05-jakarta-grpc-beta-20009.adoc (hunk 5 above)
Comment (lauracowen) [reply posts/2020-08-05-jakarta-grpc-beta-20009.adoc:88]: ah yes, got it. thx

File: posts/2020-08-05-jakarta-grpc-beta-20009.adoc (hunk 1 above)
Comment (lauracowen) [reply posts/2020-08-05-jakarta-grpc-beta-20009.adoc]: After talking to Tom Evans, I've gone with "All Open Liberty beta and GA features and functions."
